internal/sapsystem/sapsystem.go

```diff
@@ -0,0 +1,165 @@
+package sapsystem
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"regexp"
+
+	"github.com/pkg/errors"
+
+	"github.com/SUSE/sap_host_exporter/lib/sapcontrol"
+	"github.com/spf13/viper"
+
+	"github.com/trento-project/trento/internal"
+)
+
+const sapInstallationPath string = "/usr/sap"
+const sapIdentifierPattern string = "^[A-Z][A-Z0-9]{2}$" // PRD, HA1, etc
+const sapInstancePattern string = "^[A-Z]*([0-9]{2})$"   // HDB00, ASCS00, ERS10, etc
```

Comment:
You probably want that to be "^[A-Z]+([0-9]{2})$", i.e. **at least one** character in front of 2 digits.

Reply:
nice catch, Gereon!